hbase-server/src/main/java/org/apache/hadoop/hbase/ccsmap/HugeHeapChunk.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.hadoop.hbase.ccsmap;
+
+
+public class HugeHeapChunk extends OnHeapChunk {
```

Comment:
Oh, this one is redundant currently, let's not consider the huge situations and focus on general situation.

Reply:
Do you mean we will add `HugeHeapChunk` in another PR? If so, could you remove this class from this PR to make it clear? Thanks.

Reply:
Removed.